samples/caf_sensor_manager/CMakeLists.txt

```diff
@@ -5,19 +5,28 @@
 #
 cmake_minimum_required(VERSION 3.20.0)
 
+if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/boards/${BOARD}_remote.cmake")
+  include(${CMAKE_CURRENT_SOURCE_DIR}/boards/${BOARD}_remote.cmake)
+endif()
+
+set(ZEPHYR_EXTRA_MODULES ${CMAKE_CURRENT_LIST_DIR})
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project("Sensor Aggregator sample")
 
-# NORDIC SDK APP START
+if(NOT CONFIG_EVENT_MANAGER_PROXY)
```

Comment:
The problem is I cannot use in the way you suggest (**NOT** CONFIG_EVENT_MANAGER_PROXY).
I will leave it as it is.

Reply:
My point was that this should be included in the application build based on `SENSOR_MANAGER_SENSOR_SIM_CTRL` option instead of `NOT EVENT_MANAGER_PROXY`.
Afaict for nrf53 sensor sim is disabled and emp is enabled, so this would work. For nrf52 emp is disabled and sensor sim is enabled. This should work but i would not insist.
Generally i think that for `target_source*` of some subsystem X, kconfig option corelated with X should be used if file is included conditionally.